test/component_expander_test.js

```diff
@@ -0,0 +1,285 @@
+      expect($expander.find('button').first().attr('aria-expanded')).to.equal('false');
+      expect($expander.find('button').first().attr('aria-controls')).to.include('Expander_');
+    });
+
+    after(function() {
+      $('#'+STRING_EXPANDER_ID).remove();
+    });
+  });
+
+  describe("When wrap_content is true", function() {
+    it("should wrap contents with a div", function() { 
+      var $expander = $('#' + EXPANDER_ID);
+
+      expect($expander.children().length).to.equal(2);
+      expect($expander.children().last().get(0).className).to.equal(CONTAINER_CLASSNAME);
+      expect($expander.children().last().attr('id')).to.include('Expander_');
+      expect($expander.children().last().children().length).to.equal(3);
+    });
+  });
+
+  describe("When wrap_content is false", function() {
+    const UNWRAPPED_EXPANDER_ID = "unwrapped-expander-test-id";
+
+    before(function() { 
+      var $dlExpander = $(`<dl>
+        <dt>Title</dt>
+        <dd>
+          <p>Lorem ipsum dolor sit amet consecteteur adipiscing elit</p>
+          <p>Lorem ipsum dolor sit amet consecteteur adipiscing elit</p>
+        </dd>
+      </dl>`);
+
+      $dlExpander.attr('id', UNWRAPPED_EXPANDER_ID );
+      $(document.body).append($dlExpander);
+      
+      dlExpander = new Expander($dlExpander, {
+        activator: $dlExpander.find('dt').first(),
+        wrap_content: false,
+      });
+    });
+
+    it('does not wrap content with a div', function() {
+      var $expander = $('#' + UNWRAPPED_EXPANDER_ID);
+
+      expect($expander.find('div').length).to.equal(0);
+      expect($expander.find('dd').first().get(0).className).to.equal(CONTAINER_CLASSNAME);
+      expect($expander.find('dd').first().attr('id')).to.include('Expander_');
+    });
+
+    after(function() {
+      $('#' + UNWRAPPED_EXPANDER_ID).remove();
+    });
+  });
+  
+  describe("When auto_open is false", function() {
+    it("should be closed", function() {
+      var $expander = $('#' + EXPANDER_ID);
+      var instance = $expander.data('instance');
+
+      expect($expander.find('.' + CONTAINER_CLASSNAME ).get(0).style.display).to.equal('none');
+      expect($expander.find('button').first().attr('aria-expanded')).to.equal('false');
+      expect(instance.isOpen()).to.be.false;
+    })
+  });
+
+  describe("When auto_open is true", function() {
+    before(function() {
+      $expander = $('#'+EXPANDER_ID);
+      expander = new Expander($expander, {
+        activator: $expander.find('> h2').first(),
+        auto_open: true,
+      });
+    });
+
+    it("should be open", function() {
+      var $expander = $('#' + EXPANDER_ID);
+      var instance = $expander.data('instance');
+
+      expect($expander.find('.' + CONTAINER_CLASSNAME ).get(0).style.display).to.equal('');
+      expect($expander.find('button').first().attr('aria-expanded')).to.equal('true');
+      expect(instance.isOpen()).to.be.true;
+    })
+  });
+});
```

Comment:
Nice one 👍 
Have reassigned the work ticket `2509-component-expander` from my queue to you.